.gitignore

```diff
@@ -10,7 +10,6 @@ __pycache__/
 *.log
 *.out
 *.synctex.gz
-*.pdf
```

Comment:
This change is fine.

If you need to make any further changes to this PR, you might consider also changing azure-pipelines.yml to exempt .gitignore changes so that you don't have to run the full set of checks, since all of your other changes are in the doc subdirectory.